db/src/db.ts

```diff
@@ -135,14 +135,22 @@ export function buildComparator([p, direction]: Q.Order) {
   return direction === Q.ASC ? ascend(path(p)) : descend(path(p));
 }
 
+export function wrappedMatch(doc: Document, filter: Q.Filter): boolean {
```

Comment:
Cool.  Please add a comment why we need this (so I don't delete it as "unnecessary cruft").